angular-workspace/components/accordion/accordion.content.ts

```diff
@@ -203,6 +203,19 @@ export const compactSections: AccordionSection<StoryFnAngularReturnType>[] = [
   section4,
 ];
 
+export const neutralSections: AccordionSection<StoryFnAngularReturnType>[] = [
```

Comment:
Ik denk dat deze, compactSections en conclusionSections overbodig zijn. En dat waar het object gebruikt wordt ook `allowMultipleOpenSections` of `basicSectsions` gebruikt kan worden.

Reply:
Klopt, zijn allemaal precies hetzelfde als `basicSections`.